Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -425,10 +452,23 @@ extension CustomerCenterConfigData.Localization {
 
 extension CustomerCenterConfigData.HelpPath {
 
-    init(from response: CustomerCenterConfigResponse.HelpPath) {
+    init?(from response: CustomerCenterConfigResponse.HelpPath) {
```

Comment:
Note that I changed the constructor to be nullable. If we get a custom url path without a url or open method, we can filter it out and this was the easiest way to achieve that

Reply:
Hmm so for customer center I would specially take a very "lax" approach. As in, if there is something misconfigured in the backend, I would prefer to try to display as much as possible instead of failing entirely. For example in this case, it basically means that if somehow a misconfiguration happened, we would just hide the path. I think we should do this since we're still changing the schema every now and then and we should be flexible.

Having said that, we could indeed throw in the constructor and handle it when creating the paths. The code becomes a bit less legible IMO, since we need to handle the catch though... I'm not sure TBH, since I know optional constructors are actually quite common in Swift (even if I don't like them as much 😅)... Wdyt @vegaro @aboedo ?

Reply:
yeah, I think it also kinda maps better to what's happening underneath - you're using an optional constructor to try to build the URL in any case

Reply:
Right, this is essentially validating JSON, which I agree should be lax. We could avoid handling the catch and just doing `try?`: 
```swift
self.paths = response.paths.compactMap { try? CustomerCenterConfigData.HelpPath(from: $0) }
```

I don't feel super strongly about this btw, especially since optional constructors are common in Swift, as you said.

Reply:
Hmm yeah that's not bad. I also don't feel too strongly about this TBH... But considering we don't do that many throws along the codebase, I'm thinking on leaving it as is... If you feel we should better throw an error, I'm ok changing to your suggestion.

Reply:
No it's minor imo. Feel free to leave as is!

Reply:
I would also leave this as nullable as well because of the optional constructor it is using underneath